### PR TITLE
Audit Yarn

### PR DESCRIPTION
**Pull request by asafgabai**
- [x] All [tests](https://github.com/jfrog/jfrog-cli#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] All [static analysis checks](https://github.com/jfrog/jfrog-cli/actions/workflows/analysis.yml) passed.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
-----
Added Yarn support to the audit command.
Hide the deprecated audit commands (audit-mvn, audit-gradle, audit-npm, audit-go, audit-pip and audit-pipenv) from the commands list in the CLI help.

Depends on:
https://github.com/jfrog/build-info-go/pull/88
https://github.com/jfrog/jfrog-cli-core/pull/407

### CODE REVIEW

**Review by eyalbe4 — CHANGES_REQUESTED** [review]
It's important that the PR title and description include all the PR changes. If they don't, these changes will not be visible in the release notes. In this case, this PR also hides all the deprecated audit commands, which is not mentioned.

In addition, it's good that a PR description includes a link to the other PRs is depends on.

File: utils/cliutils/commandsflags.go [hunk 1]
```diff
@@ -1205,11 +1205,15 @@ var flagsMap = map[string]cli.Flag{
 	},
 	Npm: cli.BoolFlag{
 		Name:  Npm,
-		Usage: "[Optional] Request audit for a npm project.` `",
+		Usage: "[Optional] Request audit for an npm project.` `",
+	},
+	Yarn: cli.BoolFlag{
+		Name:  Yarn,
+		Usage: "[Optional] Request audit for a Yarn project.` `",
```
Comment (eyalbe4) [inline utils/cliutils/commandsflags.go:1212]: ...for a Yarn project... --> ...for a Yarn 2 project...

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: utils/cliutils/commandsflags.go (hunk 1 above)
Comment (asafgabai) [reply utils/cliutils/commandsflags.go:1212]: Changed to:
...for a Yarn 2+ project...

File: go.mod [hunk 2]
```diff
@@ -91,8 +91,8 @@ require (
...
 )
 
-// replace github.com/jfrog/jfrog-client-go => github.com/jfrog/jfrog-client-go v1.13.2-0.20220607094927-52dc3f597671
+replace github.com/jfrog/build-info-go => github.com/asafgabai/build-info-go v0.0.0-20220619100315-04b81744d42e
 
-// replace github.com/jfrog/build-info-go => github.com/jfrog/build-info-go v1.3.0
+replace github.com/jfrog/jfrog-client-go => github.com/jfrog/jfrog-client-go v1.14.1-0.20220615165226-8d3bac346a53
 
-replace github.com/jfrog/jfrog-cli-core/v2 => github.com/jfrog/jfrog-cli-core/v2 v2.16.1-0.20220616140011-e1c3549a7543
+replace github.com/jfrog/jfrog-cli-core/v2 => github.com/asafgabai/jfrog-cli-core/v2 v2.0.0-20220619124921-b4644ef5a653
```
Comment (asafgabai) [inline go.mod:98]: Reminder for myself to update this before the merge.

**Pull request merged**